themes/beaver/assets/css/components/c-hero-sections.css

```
@@ -1092,4 +1092,83 @@
   background: rgba(26, 140, 255, 1);
+}
+
+/* ==========================================================================
+   CAREERS PAGE HERO SECTION - FL-NODE EXTRACTION (22/43)
+   ========================================================================== */
+
+/* fl-node-ydac1kbu0mr5 - Careers Hero Section */
+.c-hero-section--careers > .fl-row-content-wrap,
+.fl-node-ydac1kbu0mr5 > .fl-row-content-wrap {
+  background-color: #F5F6F8;
+  background-color:#f5f6f8;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)